src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax;
+
+public class RightTriangle extends Shape {
+    private double side;
+    private double base;
+
+    public RightTriangle(String color, double side, double base) {
+        super(Colors.valueOf(color));
+        this.side = Math.round(Math.abs(side));
+        this.base = Math.round(Math.abs(base));
```

Comment:
Rounding the dimensions using `Math.round()` might not be necessary unless specified by the task requirements. Ensure this aligns with the intended functionality.